src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug, name, sex } = person;
+
+  return (
+    <NavLink to={`/people/${slug}`} className={cn({ 'has-text-danger': sex === 'f' })}>
```

Comment:
It would be cool to follow suit not to use magic values in the code, but to declare them through variables:
```
const FEMALE = 'f';
```
Now it is not necessary to fix it, but in real projects people try to write it like this. This may seem insignificant, but on large projects this approach can save you from mistakes.